### ISSUE TITLE
verifyJWT is broken

### ISSUE THREAD

**Issue by mi-xu** [uport-project/uport-js#40]
`UportLite.get` expects a callback as the second param, which it passes through to [`fetchIpfs`](https://github.com/uport-project/uport-lite/blob/23ed3bcc20be860328e40277f94c3d2cd43af392/src/index.js#L195).  However, `verifyJWT` [treats it as a promise](https://github.com/uport-project/uport-js/blob/ef4579c50ccfd0b41a938025c08e6a874783d0a9/src/JWT.js#L70) and only passes in a single issuer param, so `fetchIpfs` always fails because it attempts to call the undefined callback.

**Comment by innovatielab** [comment]
This issue was supposed to be solved in uport-connect 0.7.1, but I experience problems again with line 106-108 in JWT.js
```
if (payload.exp && payload.exp >= LEGACY_MS && payload.exp <= Date.now() || payload.iat < LEGACY_MS && payload.exp <= Date.now() / 1000) {
         return reject(new Error('JWT has expired: exp: ' + payload.exp + ' < now: ' + Date.now() / 1000));
}
```

It looks like `payload.exp` is not refreshed and keeps a value from some previous session with uPort, in my case from the day before...

**Comment by localredhead** [comment]
Hi @innovatielab, this seems to bee a separate issue from the one that was reported.  If you are still experiencing this problem can you open a new issue?  (Be sure to double check your time settings, did they change around that time?)

@mi-xu closing this for now.
